Story/Friday13th/CoreFriday13th.cs

```diff
@@ -0,0 +1,825 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class CoreFriday13th
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        Core.RunCore();
+
+        Core.SetOptions(false);
+    }
+
+    public void DoAll()
+    {
```

Comment:
needs
 if (Core.isCompletedBefore(3119) || !Core.IsMember && !CalculateFriday13())
            return;
to be added as it it is called in 00DoAllFridat13th so requires a check